MultiplayerMod/Multiplayer/Commands/Tools/CopySettings.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using MultiplayerMod.Core.Logging;
+using MultiplayerMod.Game.Context;
+using MultiplayerMod.Game.Effects;
+using MultiplayerMod.Game.Tools.Events;
+
+namespace MultiplayerMod.Multiplayer.Commands.Tools;
+
+[Serializable]
+public class CopySettings : IMultiplayerCommand {
+
+    private static readonly Core.Logging.Logger log = LoggerFactory.GetLogger<CopySettings>();
+
+    private CopySettingsEventArgs @event;
+
+    public CopySettings(CopySettingsEventArgs @event) {
```

Comment:
Should it be `arguments` as in other places? :)

Reply:
Whoops.